ghost/core/core/server/api/endpoints/posts-public.js

```diff
@@ -31,6 +31,34 @@ module.exports = {
             cacheInvalidate: false
         },
         cache: postsPublicService.api?.cache,
+        generateCacheKey(frame) {
+            const stringify = require('json-stable-stringify');
+            const keyData = {
+                include: frame.options?.include?.split(',').sort(),
+                filter: frame.options?.filter,
+                fields: frame.options?.fields?.split(',').sort(),
+                formats: frame.options?.formats?.split(',').sort(),
+                limit: frame.options?.limit,
+                order: frame.options?.order,
+                page: frame.options?.page || 1,
+                absolute_urls: frame.options?.absolute_urls,
+                collection: frame.options?.collection
+            };
+
+            if (frame.options?.context?.member) {
+                keyData.auth = {
+                    free: frame.options?.context?.member.status === 'free',
+                    tiers: frame.options?.context?.member.products?.map((product) => {
+                        return product.slug;
+                    }).sort(),
+                    labels: frame.options?.context.member.labels?.map((label) => {
```

Comment:
Do labels affect content gating?

Reply:
Nope! As discussed in slack - have removed :)